Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +15,51 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static int FindDigit(string equation)

```

Comment:
As you have differenciated each component logically i.e. retrieving the params, prrocessing them and finding the defected one and replacing it with the correct one.
It will be a good practice to make unique methods for each block. Each method shall perform a unique action. Please make those corrections.

Reply:
Created separate methods.